packages/agoric-cli/scripts/get-sdk-package-names.js

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env node
+/* global process, Buffer */
+import { spawn } from 'child_process';
+import { basename } from 'path';
+
+const ps = spawn('yarn', ['workspaces', '--silent', 'info'], {
+  stdio: ['ignore', 'pipe', 'inherit'],
+});
+
+// Get Buffers of output.
+const chunks = [];
+ps.stdout.on('data', data => chunks.push(data));
+
+// Wait for the process to exit.
+ps.on('close', code => {
+  if (code) {
```

Comment:
nit: It's poorly documented, but technically `code` can be `null` if the process exited from a signal, which in some cases indicates an abnormal termination. In this case I believe a signal terminated process should be abnormal.

```suggestion
  if (code !== 0) {
```